### PR TITLE
main/nfs-utils: upgrade to 2.3.1

### PR DESCRIPTION
**Pull request by TBK**

### CODE REVIEW

File: main/nfs-utils/APKBUILD [hunk 1]
```diff
@@ -1,11 +1,14 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=nfs-utils
-pkgver=2.1.1
-_basever=$pkgver
-pkgrel=4
+pkgver=2.2.1
+# Get the release commit hash from
+# http://git.linux-nfs.org/?p=steved/nfs-utils.git;a=summary
+_commit=a867b45e2cd82c672c878bb6cc2cfd655aa7c6b4
```
Comment (jirutka) [inline main/nfs-utils/APKBUILD]: Why? There are tags in the upstream repository…

Comment (TBK) [reply main/nfs-utils/APKBUILD]: I tried to use the release tag but could not figure out how to get the archive... :/

Comment (TBK) [reply main/nfs-utils/APKBUILD]: I figured out a way to use the tag:
http://git.linux-nfs.org/?p=steved/nfs-utils.git;h=refs/tags/nfs-utils-2-3-1;a=snapshot;sf=tgz

I will make the changes later and bump the version.

gitweb is annoying...

Comment (jirutka) [reply main/nfs-utils/APKBUILD]: Yeah, gitweb is quite bad.

**Comment by TBK** [comment]
@jirutka I have made the changes.

File: main/nfs-utils/APKBUILD [hunk 2]
```diff
@@ -14,12 +14,12 @@ makedepends="util-linux-dev libtirpc-dev libcap-dev libevent-dev
 	libnfsidmap-dev keyutils-dev lvm2-dev krb5-dev sqlite-dev
 	autoconf automake libtool bsd-compat-headers"
 subpackages="$pkgname-doc $pkgname-dbg rpcgen"
-
-source="http://downloads.sourceforge.net/nfs/$pkgname-$_basever.tar.bz2
+source="$pkgname-$pkgname-$_pkgver.tar.gz::http://git.linux-nfs.org/?p=steved/nfs-utils.git;h=refs/tags/nfs-utils-$_pkgver;a=snapshot;sf=tgz
```
Comment (ncopa) [inline main/nfs-utils/APKBUILD]: you it be an idea to use this instead of gitweb? https://www.kernel.org/pub/linux/utils/nfs-utils/

Comment (TBK) [reply main/nfs-utils/APKBUILD]: Yes, if I had known it existed xD

Comment (TBK) [reply main/nfs-utils/APKBUILD]: Third time's the charm... Switched to kernel.org.

**Pull request closed without merging**